### DISCUSSION TITLE
Can Turtle be made destructable?

### DISCUSSION THREAD

**Discussion by WellThatIsntRight** [cc-tweaked/CC-Tweaked#1251] [Q&A] [answered]
Turtles are indestructible (correct me if I'm, wrong please).
Can they be made destructible? I would prefer that.

**Comment by SquidDev** [comment, marked as answer]
> Turtles are indestructible

Do you mean with explosives or generally? Advanced turtles have a higher explosion resistance (and normal turtles are resistant to creeper explosions), but that should be their only difference from other blocks.

**Reply by WellThatIsntRight** [reply]
Generally, I think. As in a Mob wont attack it and even if one did it would be incapable of inflicting damage. I also don't believe that they can be be squished under gravel as a player mob could be. I might be wrong. I would love to be wrong.
I don't think this is something that could be changed. Just learned there's rarely harm in asking questions - maybe it is.

**Reply by SquidDev** [reply]
Ahh, to clarify, turtles are blocks rather than entities. So mobs can't really attack them deliberately.

**Reply by WellThatIsntRight** [reply]
Thanks for understanding what I wasn't able to clearly articulate.
I assume that that for reasons that are beyond the scope of this question they must remain blocks and cannot be made into entities and therefore cannot be made destructible.  
I also assume that there is no a mechanism to 'embed' an [invisible to player eyes] entity within them for the purpose of making them destructible and/or squishable?

**Reply by SquidDev** [reply]
I think this is one of those cases where _technically_ it's possible, but it's a lot of work to do (and even more to get right!).

Either way, I'm afraid it's a change I'm very unlikely to make. I think for most players probably don't want turtles dying on them - I for one definitely wouldn't want them to be squashed by gravel!

There are some mods out there which can make mobs attack/break blocks (afraid I don't have any recommendations), which might be worth looking in to. It might get you part of the way towards what you're looking for.

**Reply by WellThatIsntRight** [reply]
Understood. Thanks for your time. I didn't entirely understand what I was asking while I was asking it.
